src/main/java/com/autotune/analyzer/recommendations/RecommendationConstants.java

```diff
@@ -654,6 +654,10 @@ public static final class RecommendationValueConstants {
         public static final Double MEM_SPIKE_BUFFER_DECIMAL = 0.05;
         public static final Double DEFAULT_CPU_THRESHOLD = 0.1;
         public static final Double DEFAULT_MEMORY_THRESHOLD = 0.1;
+
+        public static final int THRESHOLD_HRS_SHORT_TERM = 6;
```

Comment:
is it worth setting this threshold from app config ? so that code change is not required.

Reply:
Yes that's a great idea, we can have an app config entry for this with which we can make user to tune it and make these entries as default values.